int64 tick_id
int64 bee_id
int64 received_reward
int64 observed_flower_id
int64 observed_reward
